contracts/projectRegistry/ProjectRegistry.sol

```diff
@@ -189,6 +211,20 @@ contract ProjectRegistry is Initializable {
         owners.count = 1;
     }
 
-    // Private functions
-    // ...
+    /**
+     * @notice Create a new round for a project
+     * @param encodedParameters Encoded parameters for round creation
+     */
+    function createRound(bytes calldata encodedParameters)
+        external onlyProjectOwner(projectID)
+        returns(IRound)
+    {
+        /// @dev Check that the project has a project
+        require(
+            projects[projectID].programMetadata.protocol != 0 &&
+            projects[projectID].programMetadata.pointer != "",
+            "PR005"
```

Comment:
^ @KurtMerbeth based on the idea that you would need programMetadata to create a round